tests/test_locales.py

```diff
@@ -2552,3 +2552,115 @@ def test_plurals_mk(self):
         assert self.locale._format_timeframe("months", 11) == "11 ай"
         assert self.locale._format_timeframe("year", 1) == "бір жыл"
         assert self.locale._format_timeframe("years", 12) == "12 жыл"
+
+
+@pytest.mark.usefixtures("lang_locale")
+class TestNorwegianLocale:
+    def test_describe(self):
+        assert self.locale.describe("now", only_distance=True) == "instantly"
```

Comment:
I think this is what is causing the error.

Reply:
Huh - how did I miss that 😅 

Updated!